modules/nw-egressnetworkpolicy-object.adoc

```diff
@@ -143,6 +145,33 @@ spec:
       protocol: TCP
     - port: 443
 ----
+
+[id="configuringNodeSelector-example_{context}"]
+== Example NodeSelector {kind}
+
+As a cluster administrator, you can create an egress firewall policy rule using `NodeSelector`.
+
+[TIP]
+====
+Rather than adding manual rules per host IP, use `NodeSelector` to create a label to allow pods behind an egress firewall to access host network pods.
+====
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {api}
+kind: {kind}
+metadata:
+  name: default
+spec:
+  egress:
+  - type: Allow
+    to:
+      NodeSelector
+----
+[TIP]
+====
+Rather than adding manual rules per host IP, use `NodeSelector` to create a label to allow pods behind an egress firewall to access host network pods.
```

Comment:
We probably want to avoid too many tips.

We're using _IP address_ rather than just _IP_ in virtually every instance in the docs.

Because this matches cluster network traffic, node is probably more appropriate than host, which is more likely to be the underlying physical system upon which the k8s node runs, along with a separate network configuration.